library/src/main/java/com/google/android/cameraview/CameraView.java

```diff
@@ -407,13 +410,15 @@ public void takePicture() {
         mImpl.takePicture();
     }
 
-    private class CallbackBridge implements CameraViewImpl.Callback {
+    private static class CallbackBridge implements CameraViewImpl.Callback {
```

Comment:
I make CallbackBridge 'static' anonymous class and pass the CameraView in the constructor.  This allows CallbackBridge to clear out any references to CameraView when the CameraView is not used anymore.  So, when the activity is destroyed, CameraView won't cause any leaks.